shared/middleware/varnish_environment.rb

```diff
@@ -85,9 +85,10 @@ def language_to_locale(language)
     end
 
     def log_ge_region_select_event(ge_region)
+      session = request.session
       Metrics::Events.log_event(
         event_name: 'Global Edition Region Selected',
-        request: request,
+        session: session,
```

Comment:
It probably makes more sense to just put `request.session` here rather than assign it to a variable above.